src/main/java/christmas/domain/event/EventBadge.java

```diff
@@ -0,0 +1,31 @@
+package christmas.domain.event;
+
+import static christmas.domain.Constant.*;
+
+public enum EventBadge {
+    NONE("없음", 0), STAR("별", 5000), TREE("트리", 10000), SANTA("산타", 20000);
+
+    private final String name;
+    private final int basePrice;
+
+    EventBadge(String name, int basePrice) {
+        this.name = name;
+        this.basePrice = basePrice;
+    }
+
+    // 정적 생성 메서드
+    public static EventBadge create(int totalBenefit) {
+        if (totalBenefit >= SANTA.basePrice) {
+            return EventBadge.SANTA;
+        } else if (totalBenefit >= TREE.basePrice) {
+            return EventBadge.TREE;
+        } else if (totalBenefit >= STAR.basePrice) {
+            return EventBadge.STAR;
+        }
+        return EventBadge.NONE;
+    }
```

Comment:
```suggestion
    public static EventBadge create(int totalBenefit) {
        for(EventBadge eventBadge : EventBadge.values()){
            if(totalBenefit >= eventBadge.basePrice){
                return eventBadge;
            }
        }
        return EventBadge.NONE;
    }
```

enum은 순서를 보장하기 때문에 큰 수부터 선언하시면 위 코드처럼 한번에 처리할 수 있어요! 이 방법은 어떠신가요?? 😄